requirements.txt

```diff
@@ -1,8 +1,8 @@
 numpy~=1.23.5
-pandas
+pandas==2.0.3
 dm-tree
 tabulate
-scikit-image
+scikit-image==0.21.0
```

Comment:
Same thing here.. but this might be used, just double check where.

Reply:
Same for this one (I checked the entrypoint for this package, it's indeed `skimage`)
![image](https://github.com/user-attachments/assets/48421381-5408-47a9-817a-b5316f0c97f0)